src/lib/plugin.js

```diff
@@ -19,7 +19,7 @@ const find = require('lodash/find')
 const backoffMin = 1000
 const backoffMax = 30000
 
-const REQUIRED_LEDGER_URLS = [ 'transfer', 'transfer_fulfillment', 'transfer_rejection', 'account', 'account_transfers', 'message' ]
+const REQUIRED_LEDGER_URLS = [ 'transfer', 'transfer_fulfillment', 'transfer_rejection', 'account', 'websocket', 'message' ]
```

Comment:
No, I think you're right, Evan. What if you are using plugin-bells in the browser (e.g. interledger/chrome-ilp-demo)? Then this will break still. Either plugin-bells should always use the auth token flow or at least when it detects that it's running in a browser.